lib/internal/bootstrap/node.js

```diff
@@ -85,6 +85,10 @@ const {
   },
 } = internalBinding('util');
 
+require('internal/process/arch'); // Ensure the process arch is cached
+require('internal/process/platform'); // Ensure the process platform is cached
```

Comment:
I don’t think it’s necessary to add internal modules for these, let  alone individual ones. You could just save these in an object that’s exported by e.g. lib/internal/process/per_thread.js (which is already in the snapshot) and load them in the navigator implementation.

Reply:
Thanks, that sounds like a great suggestion - as long as the original values are preserved, the problem is fixed. Will update to this today.